js/ui/text_box/ui.text_editor.base.js

```diff
@@ -444,8 +444,7 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
-
-    _cleanLabelObservable: function() {
+    _unobserveLabelContainerResize: function() {
         if(this._labelContainerElement) {
```

Comment:
RO does nothing if we call `unobserve` with not subscribed element as a parameter. So this condition is not required